commands/response.go

```diff
@@ -152,20 +136,21 @@ func (r *response) SetLength(l uint64) {
 	r.length = l
 }
 
-func (r *response) Error() *Error {
+func (r *response) Error() *cmdkit.Error {
 	return r.err
 }
 
-func (r *response) SetError(err error, code ErrorType) {
-	r.err = &Error{Message: err.Error(), Code: code}
+func (r *response) SetError(err error, code cmdkit.ErrorType) {
+	log.Debugf("SerError called. Stack: %s", debug.Stack())
```

Comment:
might want to either remove this message, or fix the typo

Reply:
thanks, I removed it.